src/commands/set.json

```diff
@@ -85,6 +85,20 @@
                 "name": "value",
                 "type": "string"
             },
+            {
+                "name": "comparison-value",
```

Comment:
i suppose we also need to add a since field, and update the history field.

Reply:
I missed pushing this earlier, thanks for pointing out! Updated